upi/aws/cloudformation/01_vpc.yaml

```diff
@@ -245,6 +252,8 @@ Resources:
       Tags:
       - Key: Application
         Value: !Ref "AWS::StackName"
+      - Key: Name
+        Value: !Join ["-", [!Ref InfrastructureName, "-private-", FIXME: zone?]]
```

Comment:
@cuppett: Should this FIXME be `!Select [0, Fn::GetAZs: !Ref "AWS::Region"]` or something.  I'll try that after I tear down my previous run, and we'll see if it works ;).

Reply:
> Should this FIXME be...

I figured it out (5f7554baf -> 5e34eb130).  But, ick ;).